community/knative/charts/serving/charts/monitoring/charts/prometheus/templates/kube-state-metrics-pod.yaml

```diff
@@ -0,0 +1,89 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.kubeState.deployment.name | quote }}
+  namespace: "knative-monitoring"
+  labels: {{ include "labels" . | nindent 4 }}
+    app: {{ .Values.kubeState.deployment.name | quote }}
+spec:
+  replicas: {{ .Values.kubeState.deployment.replicas }}
+  template:
+    metadata:
+      labels: {{ include "labels" . | nindent 8 }}
```

Comment:
This ends up including the chart version in spec.template.metadata.labels.chart, the chart version shouldn't be present in this selector.

Reply:
Will be fixed in next commit